### PR TITLE
Nc news 7

### PR DESCRIPTION
**Pull request by brunoFernandes21**
For the error test, I tested for when the comment has an empty object.
I thought about testing for when the comment has 1 property missing or two, but the first test passes regardless of whether the object is empty or has 1 or 2 properties... Hope this makes sense.

### CODE REVIEW

File: models/comments.model.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+exports.insertComments = (comment, article_id) => {
+  const queryString =
+    "INSERT INTO comments (author, body, article_id) VALUES($1, $2, $3) RETURNING *";
+
+  const queryValues = Object.values(comment);
```
Comment (mezzDavies) [inline models/comments.model.js]: Object.values isn't the best way to create the array as the order of the elements cannot 100% be guaranteed.Simply taking the required properties of the comment object would be safer and also make the first extra test I mention pass too 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -121,3 +121,35 @@ describe("handles all bad paths", () => {
...
+  test("400: should respond with a error message when missing required fields", () => {
+    const newComment = {}
+    return request(app)
+    .post("/api/articles/1/comments")
+    .send(newComment)
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad request")
+    })
+  })
```
Comment (mezzDavies) [inline __tests__/app.test.js:213]: There are many more test here that we could implement...

- what if the comment has some extra properties as well as the required ones? I think this could be a happy path test but you'd need to only take what is required off the request body (also see comment in model).
-  what if the article id is invalid 🍓
- what if the article id is not found
- what if the username doesn't exist

**Review by mezzDavies — CHANGES_REQUESTED** [review]
Good start that has the main functionality working but a fair bit of (mainly error handling) testing to do. Please resubmit when you've done 👍

**Review by Poonam-raj — APPROVED** [review]
Awesome work, Bruno 🚀 
Feel free to merge!
